### ISSUE TITLE
Copy note contents to clipboard

### ISSUE THREAD

**Issue by amberin** [orgzly-revived/orgzly-android-revived#125]
- I have searched for existing issues that may be the same as or related to mine.

Unless I am missing something, there is no way to select a note in Orgzly and copy its content to clipboard. This is something I find myself trying to do every now and then.

I am not sure how the UX would work. The current "copy" button copies a notebook for use inside Orgzly, and I believe it would be confusing if this button _also_ copied to clipboard. But there is already that copy button, so we can't just add a button.

Maybe add it to the three dots menu when a note is selected?

**Comment by sachac** [comment]
Being able to share a note would also be quite useful for me.

**Comment by amberin** [comment]
Of course! A "share" button is what we need! Which provides the option to copy to clipboard.

**Comment by 7ie7an** [comment]
Would be realy great!

**Comment by berteh** [comment]
+1

would love to be able to export easily part of my notes (eg to my email app), ideally:

- select 1 or more notes 
- hit "share" 
- all selected notes titles get copied to clipboard to be pasted as text (raw format would be OK), with their content, along with their children notes if expanded, recursively. 
- content & children of collapsed notes would not be exported/shared, only their title (if visible).

**Comment by berteh** [comment]
fyi: my current workaround is to open local .org file in a (plain) text editor, find the selected items and copy from there... but that's really time consuming.

**Comment by islam2hamy** [comment]
+1
